src/helpers/validators/emailValidator.js

```diff
@@ -0,0 +1,8 @@
+export function isEmailValid(email) {
+    if ( /[a-z0-9!#$%&'*+/=?^_`{|}~-]+(?:\.[a-z0-9!#$%&'*+/=?^_`{|}~-]+)*@(?:[a-z0-9](?:[a-z0-9-]*[a-z0-9])?\.)+[a-z0-9](?:[a-z0-9-]*[a-z0-9])?/.test(email)) {
+      return true;
+    } else {
+      return "You must enter a valid email";
```

Comment:
Beautiful, lets validate the form as well user shouldn't be able to send an empty email, add a unit test for this to